backend/api/prompts.py

```
@@ -33,88 +38,134 @@ class PromptTestRequest(BaseModel):
         response = await client.chat.completions.create(
             model=settings.OPENAI_MODEL,
-            messages=[
-                {"role": "user", "content": prompt_text}
-            ],
-            temperature=0.0
+            messages=[{"role": "user", "content": prompt_text}],
+            temperature=0.0,
         )
         content = response.choices[0].message.content
         return {"result": content if content else ""}
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)